ports/psoc6/main.c

```diff
@@ -86,14 +107,21 @@ int main(int argc, char **argv) {
         CY_ASSERT(0);
     }
 
+    /* \x1b[2J\x1b[;H - ANSI ESC sequence for clear screen */
+    printf("\x1b[2J\x1b[;H");
+
+    mp_hal_stdout_tx_str(MICROPY_BANNER_NAME_AND_VERSION);
+    mp_hal_stdout_tx_str("; " MICROPY_BANNER_MACHINE);
+    mp_hal_stdout_tx_str("\nUse Ctrl-D to exit, Ctrl-E for paste mode\n");
+
     setvbuf(stdin,  NULL, _IONBF, 0);
     setvbuf(stdout, NULL, _IONBF, 0);
 
-    int stack_dummy;
+    // int stack_dummy;
```

Comment:
Please, remove all commented block which are not needed. Or add some comment about what is the plan, when to uncomment, todo, etc.

Reply:
ok